linux/sound.cpp

```diff
@@ -37,18 +37,15 @@ void CSound::OpenJack ( const bool bNoAutoJackConnect, const char* jackClientNam
         serverName = "default";
     }
     qInfo() << qUtf8Printable (
-        QString ( "Connecting to jack \"%1\" instance (use the JACK_DEFAULT_SERVER environment variable to change this)." ).arg ( serverName ) );
+        QString ( "Connecting to JACK \"%1\" instance (use the JACK_DEFAULT_SERVER environment variable to change this)." ).arg ( serverName ) );
```

Comment:
We use APP_NAME for "Jamulus".  I think it would help translators to put "JACK" into a `#define` and use that throughout, too -- probably for a separate issue, though.

Reply:
Did an issue get raised for this?

Reply:
Not yet

Reply:
Done: https://github.com/jamulussoftware/jamulus/issues/1811